x-pack/plugins/security_solution/public/hosts/components/kpi_hosts/risky_hosts/index.tsx

```diff
@@ -94,7 +94,7 @@ const RiskyHostsComponent: React.FC<{
           <EuiFlexItem grow={false}>
             <EuiFlexGroup gutterSize="s">
               <EuiFlexItem>
-                <HostRiskInformationButtonIcon />
+                <RiskInformationButtonIcon riskEntity={RiskScoreEntity.host} />
```

Comment:
FWIW, this component is not used. [See comment](https://github.com/elastic/kibana/pull/142401/files#diff-f0b9bf6bc39b2600120f4200a5f835e3b2c022d05115f0e5e0e7fc6e8ff71e72R62-R67)